go/config/config_test.go

```diff
@@ -197,15 +197,60 @@ func TestServerCredentials(t *testing.T) {
 		},
 	}
 
-	for i, parameter := range parameters {
-		t.Run(fmt.Sprintf("Testing [%v]", i), func(t *testing.T) {
```

Comment:
This creates a subtest. This allows you to treat each iteration as a separate test and documents each iteration in the output.  Why remove it?